react-native/react/engine/index.js

```diff
@@ -14,47 +15,27 @@ const {
 import { Buffer } from 'buffer'
 import NativeEventEmitter from '../common-adapters/native-event-emitter'
 
-// Transport which just gives us the payload and skips really sending over the wire
-class DummyTransport extends RpcTransport {
-  constructor (writeCallback, incomingRPCCallback) {
-    super({})
-    this.writeCallback = writeCallback
-    this.set_generic_handler(incomingRPCCallback)
-  }
-
-  connect (cb) { cb() }
-  is_connected () { return true }
-  reset () { }
-  close () { }
-  get_generation () { return 1 }
-
-  unwrap_incoming_error (err) {
-    if (!err) {
-      return null
-    }
-
-    if (typeof (err) === 'object') {
-      return new EngineError(err)
-    } else {
-      return new Error(JSON.stringify(err))
-    }
-  }
-
-  _raw_write_bufs (len, buf) {
-    const buffer = Buffer.concat([new Buffer(len), new Buffer(buf)])
-    const data = buffer.toString('base64')
-    this.writeCallback(data)
-  }
+let platform = 'desktop'
```

Comment:
@chrisnojima Oops, that was left over code -- we don't need a platform var anymore, just imports .mobile or .desktop now.  Removed it.